src/site.settings.php

```diff
@@ -1,16 +1,28 @@
 <?php
 
-use Platformsh\ConfigReader\Config;
+use Platformsh\ConfigReader\Config as PlatformshConfig;
 use wearewondrous\PshToolbelt\SiteSettings;
 
 $databases = [];
 $config_directories = [];
-$settings['container_yamls'][] = $app_root . '/' . $site_path . '/services.yml';
+
+$settings['container_yamls'][] = '../services/default.services.yml';
+
+// Load service overrides
+// @DEPRECATED
+if(file_exists($app_root . '/' . $site_path . '/services.yml')) {
+  $settings['container_yamls'][] = $app_root . '/' . $site_path . '/services.yml';
+}
+
+// Load service overrides
+if(file_exists($app_root . '/' . $site_path . '/override.services.yml')) {
+  $settings['container_yamls'][] = $app_root . '/' . $site_path . '/override.services.yml';
```

Comment:
1. Do we really want to introduce a new naming here? `override` vs. `local`?
2. Please add documentation to readme.

Reply:
I always thought of local as only for local development but local as project local makes sense!

Reply:
Can you please add a new section to the documentation, how to use this, and whats the logic for that?